Automattic/jetpack
# packages/connection: Fix `@covers` directives in tests

#### Changes proposed in this Pull Request:

The PHPUnit `@covers` directive requires a correct, fully qualified
class name (it doesn't honor `namespace` or `use`) and uses `::` rather
than `->` when specifying covered methods.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check the output of the Code Coverage job for this package. Are the warnings about invalid `@covers` (as seen [here](https://github.com/Automattic/jetpack/pull/20312/checks?check_run_id=3023177851#step:16:1433)) gone?
